Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org'
 gemspec
 
-gem 'rails', '~> 3.2.0'
+#gem 'rails', '~> 3.2.0'
+gem 'sunspot_rails'
+#gem "shoulda-matchers", "~> 2.6.0"
+gem "minitest", "~> 4.0"
```

Comment:
It will not be compatible with the latest master, because we're using `test-unit` there, not `minitest`.